drivers/wifi/nrf700x/Kconfig

```diff
@@ -374,3 +374,21 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config NRF_WIFI_2G_BAND
+    bool ""
+    default y if BOARD_NRF7001
+
+config NRF_WIFI_5G_BAND
+    bool ""
+
+config NRF_WIFI_OP_BAND
+    int ""
+    default 1 if NRF_WIFI_2G_BAND
+	default 2 if NRF_WIFI_5G_BAND
+	default 0
+	help
+	  Set this option to select frequency band
+	  1 - 2.4GHz
+	  2 - 5GHz
+	  0 - 2.4GHz and 5GHz Both
```

Comment:
you would probably need 6 GHz band also

Reply:
Yes probably. will add that option also. Thanks for suggestion.